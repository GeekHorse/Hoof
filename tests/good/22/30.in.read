hello 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
ok 
new 
ok 
ok 
new 
ok 
ok 
ok 
ok   six seven eight 
ok 
