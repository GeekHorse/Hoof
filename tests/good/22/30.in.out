new right one extra done
new down one two three done
new down one two done
new in five done
new down done
new in six extra done
new down six done
new down six seven eight done
new down six seven done
out
new down fifty done
out
new down one two done
